app/views/case_logs/_tasklist.html.erb

```diff
@@ -1,7 +1,7 @@
 <ol class="app-task-list app-task-list--no-numbers">
   <% @form.all_sections.map do |section_key, section_value| %>
     <li>
-      <h2 class="app-task-list__section">
+      <h2 class="app-task-list__section govuk-heading-m">
```

Comment:
I’m not sure this is needed here, as `app-task-list__section` should set the correct size.

Reply:
…and revert this change too.

Reply:
I'm happy to change it, but it sorta make it inconsistent the way we set heading styles, as `app-task-list__section` is something we've defined

Reply:
There’s not a GDS component for the task list yet (frustratingly), but they do provide the code for it via the prototype kit and it [uses this pattern for headings](https://govuk-prototype-kit.herokuapp.com/docs/templates/task-list). When/if it becomes a component, it would likely be designed very similarly.

Reply:
Ah I see, okay! I'll revert it then!